### PR TITLE
Improve ARM Support

### PR DESCRIPTION
**Pull request by FinweVI**
The initial goal was to improve ARM Support as I'm running Gitea on a small ARM Board.

I ended up adding some other improvements:
 - Improve Gitea template for better diff
 - Download a compressed file
 - Checksum the compressed file
 - GPG Check the compressed file
 - Gitea user/group is now a system user/group
 - Stop and Backup Gitea before upgrading

### CODE REVIEW

File: tasks/create_user.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 - name: "Create Gitea Group"
   group:
     name: "{{ gitea_group }}"
+    system: "yes"
```
Comment (DO1JLR) [inline tasks/create_user.yml:5]: based on  [yamlint](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy) the recomended string for boolean values are `true` or `false`.

**Review by DO1JLR — APPROVED** [review]
minor improvements possible, but looks fine to me!

thanks @FinweVI :heart:

**Comment by thomas-maurice** [comment]
Oh wow amazing work here thank you so much @FinweVI I really appreciate it ! I'm very happy to merge once the small linting fix suggested by @DO1JLR gets applied !

Thank you very much !

File: tasks/create_user.yml (hunk 1 above)
Comment (FinweVI) [reply tasks/create_user.yml]: Agreed, but the documentation says to use `yes` and `no`: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/user_module.html

Just commited 81e2af0 that should solve this.

**Review by DO1JLR — DISMISSED** [review]
thanks :heart:

**Comment by thomas-maurice** [comment]
Perfect ! Thank you so much ! Can I just ask you to rebase on master @FinweVI and I'll merge it asap !

**Comment by FinweVI** [comment]
Should be good.
But Travis doesn't want to build apprently.

**Comment by thomas-maurice** [comment]
@FinweVI fuck travis, I am on the free plan of travis-ci.com and I'm out of credits :))))))))))))))))))

Merging this, thank you !

**Review by thomas-maurice — APPROVED** [review]

**Pull request merged**